lib/src/commons/BrowserUtils.d.ts

```diff
@@ -228,4 +228,27 @@ export declare namespace BrowserUtils {
      * @param expectedText expected alert's text
      */
     function expectAlertText(expectedText: string): void;
+    /**
+     *
+     * @param mouseButton -  {LEFT = 0, MIDDLE = 1 , RIGHT = 2}
+     */
+    function mouseButtonDown(mouseButton?: string): void;
```

Comment:
Why aren't you using the `MouseButton` enum you defined?

Reply:
Done